tests/fixtures/issues/issue-900.php

```diff
@@ -9698,272 +9486,7 @@
-            'Crawler' => false,
-            'isFake' => false,
-            'isAnonymized' => false,
-            'isModified' => false,
-            'CssVersion' => '3',
-            'AolVersion' => '0',
-            'Device_Name' => 'Windows Desktop',
-            'Device_Maker' => 'unknown',
-            'Device_Type' => 'Desktop',
-            'Device_Pointing_Method' => 'mouse',
-            'Device_Code_Name' => 'Windows Desktop',
-            'Device_Brand_Name' => 'unknown',
-            'RenderingEngine_Name' => 'WebKit',
-            'RenderingEngine_Version' => 'unknown',
-            'RenderingEngine_Maker' => 'Apple Inc',
-        ],
-        'lite' => false,
-        'standard' => true,
-    ],
-    'issue-900-GO' => [
-        'ua' => 'Mozilla/6.0 (Windows; U; Windows NT 6.0; en-US) Gecko/2009032609 (KHTML, like Gecko) Chrome/2.0.172.6 Safari/530.7',
-        'properties' => [
-            'Comment' => 'Chrome 2.0',
-            'Browser' => 'Chrome',
-            'Browser_Type' => 'Browser',
-            'Browser_Bits' => '32',
-            'Browser_Maker' => 'Google Inc',
-            'Browser_Modus' => 'unknown',
-            'Version' => '2.0',
-            'MajorVer' => '2',
-            'MinorVer' => '0',
-            'Platform' => 'WinVista',
-            'Platform_Version' => '6.0',
-            'Platform_Description' => 'Windows Vista',
-            'Platform_Bits' => '32',
-            'Platform_Maker' => 'Microsoft Corporation',
-            'Alpha' => false,
-            'Beta' => false,
-            'Win16' => false,
-            'Win32' => true,
-            'Win64' => false,
-            'Frames' => true,
-            'IFrames' => true,
-            'Tables' => true,
-            'Cookies' => true,
-            'BackgroundSounds' => false,
-            'JavaScript' => true,
-            'VBScript' => false,
-            'JavaApplets' => false,
-            'ActiveXControls' => false,
-            'isMobileDevice' => false,
-            'isTablet' => false,
-            'isSyndicationReader' => false,
-            'Crawler' => false,
-            'isFake' => false,
-            'isAnonymized' => false,
-            'isModified' => false,
-            'CssVersion' => '3',
-            'AolVersion' => '0',
-            'Device_Name' => 'Windows Desktop',
-            'Device_Maker' => 'unknown',
-            'Device_Type' => 'Desktop',
-            'Device_Pointing_Method' => 'mouse',
-            'Device_Code_Name' => 'Windows Desktop',
-            'Device_Brand_Name' => 'unknown',
-            'RenderingEngine_Name' => 'WebKit',
-            'RenderingEngine_Version' => 'unknown',
-            'RenderingEngine_Maker' => 'Apple Inc',
-        ],
-        'lite' => false,
-        'standard' => true,
-    ],
-    'issue-900-GP' => [
-        'ua' => 'Mozilla/5.0 (Windows; U; Windows NT 5.1; en-US) AppleWebKit/528.8 (KHTML, like Gecko) Chrome/2.0.156.0 Version/3.2.1 Safari/528.8',
```

Comment:
The "Version" string in this one is odd, and actually causes it to be parsed as Safari instead of Chrome with the changes I've made.